docker-compose.yml

```diff
@@ -31,13 +31,13 @@ services:
     volumes:
     - .:/openemr:ro
     - .:/var/www/localhost/htdocs/openemr:rw
-    - assetsvolume01:/var/www/localhost/htdocs/openemr/public:rw
-    - configvolume01:/var/www/localhost/htdocs/openemr/sites/default:rw
-    - configvolume02:/var/www/localhost/htdocs/openemr/interface/modules/zend_modules/config:rw
-    - logvolume01:/var/log
+    - assetsvolume:/var/www/localhost/htdocs/openemr/public/assets:rw
+    - themesvolume:/var/www/localhost/htdocs/openemr/public/themes:rw
```

Comment:
this is throwing errors in the gulp build :(
I think we are stuck going back to openemr/public :(